vortex-array/src/compute/slice.rs

```diff
@@ -66,6 +80,23 @@ pub fn slice(array: impl AsRef<Array>, start: usize, stop: usize) -> VortexResul
     Ok(sliced)
 }
 
+#[allow(dead_code)]
```

Comment:
Why allow dead? Also can you inline this function, it's never called anywhere else